web/client/utils/openlayers/projUtils.js

```diff
@@ -7,19 +7,29 @@
  */
 
 import {addProjection, Projection} from 'ol/proj';
-export default {
 
-    /**
+/**
     * function needed in openlayer for adding new projection
-    */
-    addProjections: function(code, extent, worldExtent, axisOrientation, units) {
-        addProjection(new Projection({
-            code,
-            extent,
-            worldExtent,
-            axisOrientation,
-            units
-        })
-        );
-    }
+*/
+export const addProjections = function(code, extent, worldExtent, axisOrientation) {
+    addProjection(new Projection({
```

Comment:
You forgot units, I just added some days ago

Reply:
good catch, i lost it when resolving the conflict